components/ui/card/ListItemInfoCard.tsx

```diff
@@ -0,0 +1,78 @@
+import { useColorScheme } from 'nativewind';
+import React, { ReactElement } from 'react';
+import { View, Text, TouchableOpacity } from 'react-native';
+
+import CustomImage, { ImageBase64 } from '@/components/ui/image/CustomImage';
+import { IconSize } from '@/util/IconSize';
+
+interface ListItemInfoCardProps {
+  image: ImageBase64;
+  title?: string;
+  details: string;
+  iconProps?: IconProps;
+}
+
+interface IconProps {
+  icon: ReactElement;
+  color: string;
+  darkModeColor?: string;
+  onPress?: () => void;
+}
+
+const ListItemInfoCard: React.FC<ListItemInfoCardProps> = ({
+  image,
+  title = null,
+  details,
+  iconProps = null,
+}) => {
+  const { colorScheme } = useColorScheme();
+
+  const clonedIcon = iconProps
```

Comment:
why we have that statment for iconProps where we have another statment for the same condition in line 66 ?

Reply:
fixed